docs/components/generic/_index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Generic Component"
+linkTitle: "Generic"
+childTitleEndOverwrite: "Generic Component"
+weight: 55
+type: "docs"
+description: "Use a generic component to implement a component that does not fit any of the other APIs."
```

Comment:
This is more in line with the descriptions of the other components - see components top level page:
```suggestion
description: "A component that does not fit any of the other APIs."
```